libdnf/dnf-sack.cpp

```diff
@@ -2093,6 +2093,7 @@ process_excludes(DnfSack *sack, GPtrArray *enabled_repos)
 
     libdnf::PackageSet repoIncludes(sack);
     libdnf::PackageSet repoExcludes(sack);
+    bool includesExists = false;
     
```

Comment:
I guess you didn't introduce it, but there are trailing whitespaces on this line...